install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -371,6 +371,8 @@ data:
   tunnel: "disabled"
   enable-endpoint-routes: "true"
   enable-local-node-route: "false"
+{{- else if .Values.aksbyocni.enabled }}
+  tunnel: "vxlan"
 {{- else }}
   tunnel: {{ .Values.tunnel }}
```

I know this line is unchanged by this PR, but since we're here could we `quote` please?

```suggestion
  tunnel: {{ .Values.tunnel | quote }}
```